admin/src/main/resources/io/buoyant/admin/js/src/router_service.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+define([
+  'jQuery',
+  'template/compiled_templates',
+  'bootstrap'
+], function($,
+  templates
+) {
```

Comment:
this argument style is kinda wonky, either `$` should get its own line or all args on one line.